### PR TITLE
[16.0][MIG] rma_sale

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Migration to 16.0

Depends on:

* [x] rma_account #397 

@ForgeFlow

### CODE REVIEW

File: rma_sale/tests/test_rma_stock_account_sale.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                            "price_unit": 800,
+                        },
+                    ),
+                ],
+                "pricelist_id": cls.env.ref("product.list0").id,
+            }
+        )
+        cls.so1.action_confirm()
+        for ml in cls.so1.picking_ids.move_line_ids:
+            ml.qty_done = ml.product_uom_qty
```
Comment (florian-dacosta) [inline rma_sale/tests/test_rma_stock_account_sale.py]: field has been renamed `reserved_uom_qty`

File: rma_sale/tests/test_rma_stock_account_sale.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        )
+        rma_line.partner_id = self.so1.partner_id
+        rma_line.sale_line_id = so_line
+        rma_line.price_unit = 4356
+        rma_line = rma_line.save()
+        rma_line.action_rma_to_approve()
+        picking = self._receive_rma(rma_line)
+        # The price is not the standard price, is the value of the outgoing layer
+        # of the SO
+        rma_move_value = picking.move_lines.stock_valuation_layer_ids.value
```
Comment (florian-dacosta) [inline rma_sale/tests/test_rma_stock_account_sale.py]: `move_lines` has been renamed `move_ids`

File: rma_sale/tests/test_rma_stock_account_sale.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        so_move_value = self.so1.picking_ids.mapped(
+            "move_lines.stock_valuation_layer_ids"
+        )[-1].value
```
Comment (florian-dacosta) [inline rma_sale/tests/test_rma_stock_account_sale.py:71]: same here `move_lines` renamed `move_ids` also no need to use `mapped` here anymore

File: rma_sale/tests/test_rma_stock_account_sale.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        self.assertEqual(rma_move_value, -so_move_value)
+        # Test the accounts used
+        account_move = picking.move_lines.stock_valuation_layer_ids.account_move_id
```
Comment (florian-dacosta) [inline rma_sale/tests/test_rma_stock_account_sale.py]: same here `move_lines` => `move_ids`

**Comment by DavidJForgeFlow** [comment]
Hi @florian-dacosta, thank you for your review! I've updated the requested changes.

**Pull request merged**